addons/mail/static/src/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -40,6 +37,10 @@
     height: 34px;
 }
 
+.o_ChatterTopbar_followButton {
+    display: flex;
```

Comment:
In fact, the inner button was not taking all the available place. Making it flex make it stretch by default and then fill all the available space.

Reply:
Ok then it really looks like we want this on `o_FollowButton` instead, so that it can take the available space no matter from which component it is included from.